models/user/user.go

```diff
@@ -1270,3 +1270,10 @@ func IsUserVisibleToViewer(ctx context.Context, u, viewer *User) bool {
 	}
 	return false
 }
+
+func GetOrderBy() string {
+	if setting.UI.DefaultShowFullName {
+		return "full_name"
```

Comment:
```suggestion
		return "full_name, name"
```

I think it should be like this. Otherwise L-G-T-M